src/common/com/intellij/plugins/haxe/model/HaxeImportModel.java

```diff
@@ -0,0 +1,154 @@
+  @Override
+  public List<HaxeModel> getExposedMembers() {
+    FullyQualifiedInfo qualifiedInfo = getQualifiedInfo();
+    List<HaxeModel> result = null;
+    if (hasWildcard()) {
+      if (qualifiedInfo.memberName != null) return Collections.emptyList();
+
+      if (qualifiedInfo.fileName != null && qualifiedInfo.className == null) {
+        qualifiedInfo = new FullyQualifiedInfo(qualifiedInfo.packagePath, qualifiedInfo.fileName, qualifiedInfo.fileName, null);
+      }
+      List<HaxeModel> items = HaxeProjectModel.fromElement(basePsi).resolve(qualifiedInfo, basePsi.getResolveScope());
+      if (items != null && items.size() > 0) {
+        result = items.stream()
+          .filter(model -> model instanceof HaxeExposableModel)
+          .flatMap(model -> ((HaxeExposableModel)model).getExposedMembers().stream())
+          .collect(Collectors.toList());
+      }
+    }
+    else {
+      if (hasAlias() && qualifiedInfo.fileName != null && qualifiedInfo.className == null) {
+        qualifiedInfo = new FullyQualifiedInfo(qualifiedInfo.packagePath, qualifiedInfo.fileName, qualifiedInfo.fileName, null);
+      }
+      result = HaxeProjectModel.fromElement(basePsi).resolve(qualifiedInfo, basePsi.getResolveScope());
+      if (result != null && !result.isEmpty() && (result.get(0) instanceof HaxeFileModel || result.get(0) instanceof HaxePackageModel)) {
+        result = ((HaxeExposableModel)result.get(0)).getExposedMembers();
+      }
+    }
+
+    return result != null ? result : Collections.emptyList();
+  }
+
+  @Nullable
+  public PsiElement exposeByName(String name) {
+    if (name == null) return null;
+
+    if (hasWildcard()) {
+      for (HaxeModel exposedMember : getExposedMembers()) {
+        if (Objects.equals(exposedMember.getName(), name)) return exposedMember.getBasePsi();
+      }
+    }
+    else {
+      if (getReferenceExpression() != null) {
+        FullyQualifiedInfo qualifiedInfo = getQualifiedInfo();
+
+        if (equalsToAlias(name) || qualifiedInfo.equalsToNamedPart(name) || getExposedMember(name) != null) {
+          HaxeModel result = getExposedMember(name);
+          return result != null ? result.getBasePsi() : null;
+        }
+      }
+    }
+
+    return null;
+  }
+
+  private boolean equalsToAlias(String name) {
+    return hasAlias() && Objects.equals(getAliasName(), name);
+  }
+
+  private HaxeModel getExposedMember(String name) {
+    List<HaxeModel> members = getExposedMembers();
+    if (members.isEmpty()) return null;
+    if (hasAlias()) {
+      return (Objects.equals(getAliasName(), name)) ? members.get(0) : null;
+    }
+    return members.stream()
+      .filter(model -> model.getName().equals(name))
+      .findFirst()
+      .orElse(null);
+  }
+
+  @Override
+  public String getName() {
+    return null;
+  }
+
+  @Override
+  public PsiElement getBasePsi() {
+    return this.basePsi;
+  }
+
+  @Nullable
+  @Override
+  public HaxeExposableModel getExhibitor() {
+    return null;
```

Comment:
Should an import's exhibitor be the package?

Reply:
Nope, imports don't have a container that exposes them, it's just declaration. We even don't need to know who exposes them - it's useless information.